src/test/plugins/mvcc_delete_plugin_system_test.cpp

```diff
@@ -245,7 +245,7 @@ TEST_F(MvccDeletePluginSystemTest, CheckPlugin) {
       // To increase the global _last_commit_id, we need to execute a transaction with read-write operators
       // We perform some dummy updates so that the table is unmodified and the validation routine does not complain
       auto pipeline =
-          SQLPipelineBuilder{std::string{"UPDATE " + _t_name_test + " SET number = number WHERE number = 0"}}
+          SQLPipelineBuilder{std::string{"UPDATE " + _t_name_test + " SET number = number WHERE number = -1"}}
```

Comment:
Previously, this could have a row-level conflict with the logical delete, leading to 

```
../../../src/test/plugins/mvcc_delete_plugin_system_test.cpp:270: Failure
Expected: (attempts_remaining) > (-1), actual: -1 vs -1
```